### PR TITLE
[CI] More cleanup for the lint task

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

**Comment by aelovikov-intel** [comment]
Tested via https://github.com/intel/llvm/actions/runs/5381209434 + manual monitoring of the filesystem on the runner.

**Review by cperkinsintel — APPROVED** [review]

File: devops/actions/cleanup/cleanup.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 const path = process.env.GITHUB_WORKSPACE + '/*';
 console.log('Cleaning ' + path)
 require('child_process').execSync('rm -rf ' + path);
+require('child_process').execSync('rm -rf ' + process.env.GITHUB_WORKSPACE + '/.git');
```
Comment (bader) [inline devops/actions/cleanup/cleanup.js:4]: Don't we want to remove all hidden files/directories and not just `.git`?

Comment (bader) [reply devops/actions/cleanup/cleanup.js:4]: using `const path = process.env.GITHUB_WORKSPACE + '/{*,.*}';` at line 1 should work.

Comment (aelovikov-intel) [reply devops/actions/cleanup/cleanup.js:4]: I will prepare a PR with this tomorrow - want to give https://github.com/intel/llvm/pull/10074 more time to expose any potential issues post-commit on its own (although I hope that none would appear, of course).

Comment (aelovikov-intel) [reply devops/actions/cleanup/cleanup.js:4]: https://github.com/intel/llvm/pull/10095

**Pull request merged**